packages/front-end/components/Features/SDKConnections/SDKConnectionsList.tsx

```diff
@@ -40,17 +43,79 @@ export default function SDKConnectionsList() {
     project
   );
 
+  const useNewEmptyStateLayout = useFeatureIsOn(
+    "sdk-connections-new-empty-state"
```

Comment:
@jdorn useFeatureIsOn doesn't seem to respect the list of AppFeatures. Is that a problem? It's certainly more convenient for the engineer but it's out of line with using, say, `growthbook.isOn`?

Reply:
`useFeatureIsOn` is a generic method that accepts AppFeatures as a type argument.  Exactly the same as `useGrowthBook`.  Neither is particularly good at enforcing type safety because it's easy to forget to pass it in.

Reply:
Ah, I see. Thanks.